cfme/fixtures/utility_vm.py

```diff
@@ -16,77 +16,54 @@
 ping, but the required service may not be up and ready.
 """
 import os.path
-import time
-from contextlib import contextmanager
 
 import pytest
 
 from cfme.base.credential import SSHCredential
 from cfme.utils.conf import cfme_data
 from cfme.utils.generators import random_vm_name
 from cfme.utils.log import logger
-from cfme.utils.net import net_check
+from cfme.utils.net import pick_responding_ip
 from cfme.utils.ssh import SSHClient
 from cfme.utils.virtual_machines import deploy_template
-from cfme.utils.wait import TimedOutError
 
 
-def _trying_ips(vm, attempts=60, interval=10):
-    for attempt in range(attempts):
-        for ip in getattr(vm, 'all_ips', []):
-            yield ip
-            return
-        time.sleep(interval)
-
-
-@contextmanager
-def connect_ssh(vm, creds):
-    for ip in _trying_ips(vm):
-        try:
-            with SSHClient(hostname=ip, username=creds.principal, password=creds.secret) as client:
-                logger.info("SSH connected to IP %s", ip)
-                result = client.run_command("true")
-                if not result.success:
-                    raise Exception(f"Command `true` failed on ip {ip}.")
-                yield client
-                return
-        except Exception as ex:
-            logger.info("Failed to connect with SSH to %s: %s", ip, ex)
-    else:
-        raise TimedOutError(f"Coudln't find an IP responding to ssh for vm {vm}")
-
-
-def _pick_responding_ip(vm, port):
-    for ip in _trying_ips(vm):
-        if net_check(port, ip):
-            return ip
-    else:
-        raise TimedOutError(f"Coudln't find an IP of vm {vm} with port {port} responding")
+ATTEMPT_TIMEOUT = 10
+""" How long to wait until connection is determined as not successful. """
+IP_PICK_TIMEOUT = 600
+""" How many rounds to connect all the vm IPs. """
+ROUNDS_DELAY = 10
+""" How long to delay after unsucessful connection round. """
```

Comment:
```suggestion
# How long to wait until connection is determined as not successful.
ATTEMPT_TIMEOUT = 10
# How many rounds to connect all the vm IPs.
IP_PICK_TIMEOUT = 600
# How long to delay after unsuccessful connection round.
ROUNDS_DELAY = 10
```

Reply:
This wouldn't be an improvement. These are docstrings. They are supposed to be under the defined identifier, documenting it.

Reply:
Gotcha, I didn't know that these would show up as sphinx documented variables. I just had never seen a docblock for a variable.